<ui version="4.0" >
 <class>HelpDialog</class>
 <widget class="QDialog" name="HelpDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTextBrowser" name="textBrowser" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
